diff --git a/Makefile b/Makefile
index 561978f..e4e3684 100644
--- a/Makefile
+++ b/Makefile
@@ -266,15 +266,18 @@ native: dep_mg
 		-DCMAKE_OSX_SYSROOT="$(SDKPATH)" \
 		-DCMAKE_OSX_ARCHITECTURES=arm64 \
 		-DCMAKE_OSX_DEPLOYMENT_TARGET=14.0 \
-		-DCMAKE_C_FLAGS="-arch arm64" \
+		-DCMAKE_C_FLAGS="-arch arm64 -target arm64-apple-ios14.0-simulator" \
+		-DCMAKE_CXX_FLAGS="-arch arm64 -target arm64-apple-ios14.0-simulator" \
+		-DCMAKE_SHARED_LINKER_FLAGS="-target arm64-apple-ios14.0-simulator" \
+		-DCMAKE_EXE_LINKER_FLAGS="-target arm64-apple-ios14.0-simulator" \
 		-DCONFIG_BRANCH="$(BRANCH)" \
 		-DCONFIG_COMMIT="$(COMMIT)" \
 		-DCONFIG_RELEASE=$(RELEASE) \
 		..
 
-	cmake --build $(WORKINGDIR) --config $(CMAKE_BUILD_TYPE) -j$(JOBS)
+	cmake --build $(WORKINGDIR) --config $(CMAKE_BUILD_TYPE) -j$(JOBS) --target AngelAuraAmethyst awt_xawt audio_capture AmethystAccountJNI ltw
 	#	--target awt_headless awt_xawt libOSMesaOverride.dylib tinygl4angle AngelAuraAmethyst
-	rm $(WORKINGDIR)/libawt_headless.dylib
+	rm -f $(WORKINGDIR)/libawt_headless.dylib
 	echo '[Amethyst v$(VERSION)] native - end'
 
 java:
diff --git a/Natives/CMakeLists.txt b/Natives/CMakeLists.txt
index d6bf937..c1e4d1d 100644
--- a/Natives/CMakeLists.txt
+++ b/Natives/CMakeLists.txt
@@ -11,7 +11,7 @@ endif()
 
 configure_file(${CMAKE_CURRENT_LIST_DIR}/config.h.in ${CMAKE_CURRENT_LIST_DIR}/config.h)
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -target arm64-apple-ios -mcpu=apple-a8 -fcommon -fobjc-arc -ObjC -F'${CMAKE_OSX_SYSROOT}/System/Cryptexes/OS/System/Library/Frameworks'")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mcpu=apple-a8 -fcommon -fobjc-arc -ObjC -F'${CMAKE_OSX_SYSROOT}/System/Cryptexes/OS/System/Library/Frameworks'")
 set(CMAKE_C_FLAGS_DEBUG "-funwind-tables -g")
 
 if("${CMAKE_HOST_SYSTEM_NAME}" MATCHES "Linux|^GNU$|Android")
